languages/csharp/exercises/concept/properties/.meta/Example.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+
+public enum Units
+{
+    Pounds,
+    Kilograms
+}
+public class WeighingMachine
+{
+    private const decimal POUNDS_PER_KILOGRAM = 2.20462m;
+    private decimal inputWeight;
+
+    public Units Units { get; set; } = Units.Kilograms;
+    public decimal InputWeight
+    {
+        get { return inputWeight; }
+        set
+        {
+            if (value < 0)
+            {
+                throw new ArgumentException("weight cannot be negative");
```

Comment:
Will this exercise rely on a pre-req of `exceptions`? Or are we going to teach `exceptions` in this exercise?

Reply:
Nope, exceptions are a separate exercise and it should be a prerequisite.

Reply:
If you want to avoid the dependency we could do something cheesy like saying if a negative value is entered it is reverted to zero.